internal/command/project_grant_model.go

```diff
@@ -114,30 +114,44 @@ func (wm *ProjectGrantWriteModel) Query() *eventstore.SearchQueryBuilder {
 type ProjectGrantPreConditionReadModel struct {
 	eventstore.WriteModel
 
+	ResourceOwner    string
```

Comment:
the `ResourceOwner` already exists on the `WriteModel`

Reply:
removed ResourceOwner and added tests to unit tests for unexpected scenario